### PR TITLE
feat(lb): improve backend help in case healthcheck is missing

### PR DESCRIPTION
**Pull request by yfodil**

### CODE REVIEW

File: internal/namespaces/lb/v1/custom_backend.go [hunk 1]
```diff
@@ -501,7 +503,44 @@ func interceptBackend() core.CommandInterceptor {
...
-			return nil, err
+			type rawBodyResponse struct {
+				Details []struct {
+					Field   string `json:"field"`
+					Message string `json:"message"`
+					Reason  string `json:"reason"`
+				} `json:"details"`
+				Message string `json:"message"`
+				Type    string `json:"type"`
+			}
```
Comment (quantumsheep) [inline internal/namespaces/lb/v1/custom_backend.go:514]: Is there no existing structures like that from Scaleway Go SDK ? Seems like the standard structure for attribute errors.

File: internal/namespaces/lb/v1/custom_backend.go [hunk 2]
```diff
@@ -501,7 +503,44 @@ func interceptBackend() core.CommandInterceptor {
...
+			if err != nil {
+				return nil, err
+			}
+
+			switch data.Details[0].Field {
+			case "health_check":
+				return "", &core.CliError{
+					Err: fmt.Errorf("missing required 'health-check.port' and 'health-check.check-max-retries' arguments"),
+				}
+			case "Port":
```
Comment (quantumsheep) [inline internal/namespaces/lb/v1/custom_backend.go:533]:
```suggestion
			case "port":
```
Is this really in PascalCase or is it a mistake ?

File: internal/namespaces/lb/v1/custom_backend.go [hunk 3]
```diff
@@ -501,7 +503,44 @@ func interceptBackend() core.CommandInterceptor {
...
+				return "", &core.CliError{
+					Err: fmt.Errorf("missing required 'health-check.port' argument"),
+				}
+			case "CheckMaxRetries":
```
Comment (quantumsheep) [inline internal/namespaces/lb/v1/custom_backend.go:537]:
```suggestion
			case "check_max_retries":
```
Is this really in PascalCase or is it a mistake ?

**Review by quantumsheep — CHANGES_REQUESTED** [review]

File: internal/namespaces/lb/v1/custom_backend.go (hunk 3 above)
Comment (yfodil) [reply internal/namespaces/lb/v1/custom_backend.go]: Yes its PascalCase: {"details":[{"argument_name":"Port","help_message":"required got 0","reason":"constraint"},{"argument_name":"CheckMaxRetries","help_message":"required got 0","reason":"constraint"}],"message":"invalid argument(s)","type":"invalid_arguments"}

**Review by quantumsheep — APPROVED** [review]

**Pull request merged**